src/main/java/net/snowflake/ingest/streaming/internal/EpInfo.java

```diff
@@ -17,6 +21,30 @@ class EpInfo {
     this.columnEps = columnEps;
   }
 
+  /** Some basic verification logic to make sure the EP info is correct */
+  public void verifyEpInfo() {
```

Comment:
We could have more here: min is always smaller than max (both int and string), neither min nor max strings are longer than 32 bytes (in hex),

Reply:
We could add these later as we go, will merge this first to have a new release